scripts/packages/VSCodeServer/src/eval.jl

```diff
@@ -117,10 +117,12 @@ function repl_runcode_request(conn, params::ReplRunCodeRequestParams)
         show_code = params.showCodeInREPL
         show_result = params.showResultInREPL
         show_error = params.showErrorInREPL
+        try
+            JSONRPC.send_notification(conn_endpoint[], "repl/starteval", nothing)
+        catch err
+            @debug "Could not send repl/starteval notification."
```

Comment:
These won't show up in crash reporting, right?

Reply:
Yes. It's really unlikely we'd get anything useful out of this even if it did though.